src/backend/bootstrap/app.php

```diff
@@ -78,4 +78,5 @@
         // will run on the last day of the month
         $schedule->command(MailApplianceDebtsCommand::class)->weeklyOn(1, '6:00');
     })
+    ->withProviders(require __DIR__.'/providers.php')
```

Comment:
How did you find these? I'm reading the documentation and I cannot find any trace of it saying that we need to specifically load `providers.php` 😖

Reply:
The few examples, that I do find use 

```
->withProviders()
```

Would this work for us as well? I think our bootstrap file is in the default location.

Reply:
Removed the method and it still works as expected.

Reply:
True, a bunch of plugins we no longer support are still hanging around.